playwright/example.spec.ts

```
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)